cmd/frps/root.go

```diff
@@ -37,29 +37,30 @@ var (
 	cfgFile     string
 	showVersion bool
 
-	bindAddr          string
-	bindPort          int
-	bindUdpPort       int
-	kcpBindPort       int
-	proxyBindAddr     string
-	vhostHttpPort     int
-	vhostHttpsPort    int
-	dashboardAddr     string
-	dashboardPort     int
-	dashboardUser     string
-	dashboardPwd      string
-	assetsDir         string
-	logFile           string
-	logWay            string
-	logLevel          string
-	logMaxDays        int64
-	token             string
-	authTimeout       int64
-	subDomainHost     string
-	tcpMux            bool
-	allowPorts        string
-	maxPoolCount      int64
-	maxPortsPerClient int64
+	bindAddr              string
+	bindPort              int
+	bindUdpPort           int
+	kcpBindPort           int
+	proxyBindAddr         string
+	vhostHttpPort         int
+	vhostHttpsPort        int
+	dashboardAddr         string
+	dashboardPort         int
+	dashboardUser         string
+	dashboardPwd          string
+	assetsDir             string
+	logFile               string
+	logWay                string
+	logLevel              string
+	logMaxDays            int64
+	token                 string
+	authTimeout           int64
+	subDomainHost         string
+	tcpMux                bool
+	allowPorts            string
+	maxPoolCount          int64
+	maxPortsPerClient     int64
+	ResponseHeaderTimeout int64
```

Comment:
Use lowercase letter at the beginning and the word `ResponseHeaderTimeout` is not clear for others to figure out where it's used.
`vhost_http_timeout` is better.

Reply:
OK, I've changed it